packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -391,6 +500,18 @@ async function getTestResponse({
 				}
 			);
 		},
+		waitUntil: (async (promise: Promise<unknown>) => {
+			spies.waitUntil.push(promise);
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		}) as any,
```

Comment:
```suggestion
		waitUntil: async (promise: Promise<unknown>) => {
			spies.waitUntil.push(promise);
		},
```

Don't think you need that `any`? My editor isn't complaining, at least :)

Reply:
Ah thank you. I was moving fast there and meant to come back to that